tests/test_gaap.py

```diff
@@ -268,6 +268,58 @@ def testFlags(self, sigmas=[2.5, 3.0, 4.0], scalingFactors=[1.15, 1.25, 1.4]):
         self.assertTrue(record[algName + "_flag_edge"])
         self.assertFalse(record[algName + "_flag"])
 
+    @lsst.utils.tests.methodParameters(noise=(0.001, 0.01, 0.1))
+    def testMonteCarlo(self, noise, recordId=1, sigmas=[3.0, 4.0], scalingFactors=[1.1, 1.15, 1.2, 1.3, 1.4]):
+        """Test GAaP flux uncertainties.
+
+        This test should demonstate that the estimated flux uncertainties agree
+        with those from Monte Carlo simulations.
+
+        Parameters
+        ----------
+        noise : `float`
+            The RMS value of the Gaussian noise field divided by the total flux
+            of the source.
+        recordId : `int`, optional
+            The source Id in the test dataset to measure.
+        sigmas : `List` [`float`], optional
+            The list of sigmas (in pixels) to construct the `GaapFluxConfig`.
+        scalingFactors : `List` [`float`], optional
+            The list of scaling factors to construct the `GaapFluxConfig`.
+        """
+        gaapConfig = lsst.meas.extensions.gaap.GaapFluxConfig(sigmas=sigmas, scalingFactors=scalingFactors)
+        gaapConfig.scaleByFwhm = True
+
+        algorithm, schema = self.makeAlgorithm(gaapConfig)
+        # Make a noiseless exposure and measurements for reference
+        exposure, catalog = self.dataset.realize(0.0, schema)
+        record0 = catalog[recordId]
```

Comment:
record0 seems like a misnomer if the default recordId is 1 and it can be anything.

Reply:
Hmm, the 0 in `record0` was meant to denote that this is a record from a noiseless image, to be later compared with record with some pixel noise in it. Would you suggest r`ecordNoiseless` or something more verbose?

Reply:
recordNoiseless works for me, I thought the 0 referred to an index so the noiseless bit is a bit improvement. Would make it much easier to see what is going on.